EPAM_iOS_School_Practice_2_Star_Wars/EPAM_iOS_School_Practice_2_Star_Wars/Networking/SearchService.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SearchService.swift
+//  EPAM_iOS_School_Practice_2_Star_Wars
+//
+//  Created by Maxim on 27.02.2020.
+//  Copyright © 2020 Maksim Svyatov. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+protocol SearchServiceProtocol {
+    func searchPeople(for name: String, completion: @escaping ((People?, Error?)->()))
```

Comment:
Use `Result<People, Error>` in completion
https://medium.com/@azamsharp/understanding-result-type-in-swift-5-387d5ef9f45e

Reply:
Fixed